### PR TITLE
MVP Release Candidate Prod

### PR DESCRIPTION
**Pull request by MrSimonEmms**
## Ticket Number

N/A

## Description of change

Prepare a release candidate of the MVP for deployment to Prod.

Supersedes #762 #700 and #555

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

**Comment by MrSimonEmms** [comment]
FYI - this updates the LPA Questionnaire and Appeals Reply Service to latest. The DNS record hasn't yet been set up so it won't actually go out

File: releases/prod/functions.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        repository: pinscommonukscontainers3887default.azurecr.io/ping
+        tag: 1.0.0
+      config:
+        ping:
+          url: http://10.224.161.68:8001/status
+
+    functions:
+      horizon-add-document:
+        image: pinscommonukscontainers3887default.azurecr.io/horizon-add-document
+        tag: latest
```
Comment (Fenisay) [inline releases/prod/functions.yml:43]: Double check that the latest is on purpose

**Review by danWatkinson — COMMENTED** [review]
Looking at https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/763/files should I be concerned that the env-vars relating to the storage container needed to be applied to preprod but don't feature here?

**Comment by MrSimonEmms** [comment]
> Looking at https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/763/files should I be concerned that the env-vars relating to the storage container needed to be applied to preprod but don't feature here?

@danWatkinson how do you mean? They were removed in the PreProd PR. In the Prod, they never existed in the functions.yaml so they won't appear.

If I've misunderstood, please elucidate further

**Review by Russ-Clarke — APPROVED** [review]

**Pull request merged**